examples/analog_out/gen_voltage_wfm_int_clk.py

```diff
@@ -0,0 +1,17 @@
+"""Example of analog output voltage generation.
+
+This example demonstrates how to output a finite number of
+voltage samples to an Analog Output Channel using an internal
+sample clock.
+"""
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType
+
+with nidaqmx.Task() as task:
+    task.ao_channels.add_ao_voltage_chan("Dev1/ao0")
+    task.timing.cfg_samp_clk_timing(1000, sample_mode=AcquisitionType.FINITE, samps_per_chan=1000)
```

Comment:
since we're doing so few samples, I'd slow this down

```suggestion
    task.timing.cfg_samp_clk_timing(10.0, sample_mode=AcquisitionType.FINITE, samps_per_chan=1000)
```

Reply:
Alternately, we can use math to generate more samples. From `MultVoltUpdates-IntClk.c`:
```
	for(;i<4000;i++)
		data[i] = 5.0*(double)i/4000.0;
```
From `ContGen-IntClk.c` (but I see, you added a sine wave to the continuous example already):
```
	for(;i<1000;i++)
		data[i] = 9.95*sin((double)i*2.0*PI/1000.0);
```

Reply:
Okay, I take @bkeryan 's opinion, referring `MultVoltUpdates-IntClk.c` to generate 1000 samples.